analytics/policy/tests/unit/test_registration_number_policies.py

```diff
@@ -31,30 +16,37 @@ def test_invalid_pattern():
         "24-1601a8",
         "124-160188",
         "q4-160188",
-        "4-160188",  
+        "4-160188",
+        "123",
+        "",
+        None
     ]
     for registration_number in registration_numbers:
         registration_number_policies = RegistrationNumberPolicies(
             registration_number
         )
-        assert registration_number_policies.valid_pattern() is False
+        assert registration_number_policies.valid_registration_number_policy() is False
 
-def test_valid_registration_number(mock_http_response_200):
+def test_unique_registration_number_policy():
```

Comment:
please include test cases for registration_number = "", None

Reply:
These two are tested in `test_invalid_pattern`

Reply:
yeah, but the unique and existed policies should be robust enough to handle these